doc/md/fundamentals/2-types/3-functions.md

```diff
@@ -31,8 +31,9 @@ Motoko provides different types of functions based on where in the program they
 | Keyword  | Function  |
 |-------------|--------------|
 | `public`    | Makes a function callable by users or other [canisters](https://internetcomputer.org/docs/building-apps/essentials/canisters). |
```

Comment:
```suggestion
```
This isn't a property of functions per se but of the  declarations withing actor, modules, objects etc.